.github/actions/git-diff-on-components/src/index.js

```diff
@@ -95,6 +95,17 @@ function includesVersion(contents) {
   return contents.match(new RegExp(/version: "\d+\.\d+\.\d+"/g)) || contents.match(new RegExp(/"version": "\d+\.\d+\.\d+"/g));
 }
 
+function getVersion(contents) {
+  return includesVersion(contents)[0].match(/\d{3}\.\d{3}\.\d{3}/)[0];
```

Comment:
Can you check this regexp as mentioned in [this previous conversation](https://github.com/PipedreamHQ/pipedream/pull/6295#discussion_r1186457476)?

The way it is now matches exactly 3 digits for each version number, which I believe is not the desired behavior.